tools/tpm2_duplicate.c

```diff
@@ -115,16 +115,16 @@ bool tpm2_tool_onstart(tpm2_options **opts) {
 
     const struct option topts[] = {
       { "auth-key",              required_argument, NULL, 'p'},
-      { "inner-wrapper-alg",     required_argument, NULL, 'g'},
-      { "duplicate-key-private", required_argument, NULL, 'r'},
+      { "inner-wrapper-alg",     required_argument, NULL, 'G'},
+      { "privfile",              required_argument, NULL, 'r'},
```

Comment:
@williamcroberts we have called this simply "private" elsewhere.

Reply:
possibly. That's fine by me. We just need to make sure we mark this in our audit so we are consistent everywhere.